src/devices_models/devices/common/add_variable.jl

```diff
@@ -52,7 +6,11 @@ function add_variables!(
     ::Type{T},
     devices::Union{Vector{U}, IS.FlattenIteratorWrapper{U}},
 ) where {T <: VariableType, U <: PSY.Component}
-    _add_variables!(psi_container, devices, AddVariableSpec(T, U, psi_container))
+    add_variable!(
+        psi_container,
+        T(),
```

Comment:
Is it better to evaluate here instead of passing the type? I don't understand the reason for doing that instead of passing `T`. The code style in PSI has used T as type so far.